src/Features/CSharp/Portable/Completion/CompletionProviders/OperatorsAndIndexer/UnnamedSymbolCompletionProvider.cs

```diff
@@ -106,6 +107,13 @@ public override async Task ProvideCompletionsAsync(CompletionContext context)
             var document = context.Document;
             var position = context.Position;
 
+            // Escape hatch feature flag to let us disable this feature remotely if we run into any issues with it, 
+            var workspace = document.Project.Solution.Workspace;
+            var experimentationService = workspace.Services.GetRequiredService<IExperimentationService>();
+            var disabled = experimentationService.IsExperimentEnabled(WellKnownExperimentNames.UnnamedSymbolCompletionDisabled);
```

Comment:
❗ This needs an option as well so it can be manually enabled via vsregedit. Here's an example from another feature:

https://github.com/dotnet/roslyn/blob/6da1274c9d24c2f90a48290394a951b23617f2a3/src/Workspaces/Remote/Core/RemoteHostOptions.cs#L59-L60